engine/core/src/main/java/org/datacleaner/job/runner/RowProcessingPublisher.java

```diff
@@ -509,12 +508,29 @@ public void runRowProcessing(Queue<JobAndResult> resultQueue, TaskListener finis
 
         final RowProcessingMetrics rowProcessingMetrics = getRowProcessingMetrics();
         final RunRowProcessingPublisherTask runTask = new RunRowProcessingPublisherTask(this, rowProcessingMetrics);
+
         if (_parentPublisher == null) {
             final TaskListener initFinishedListener = new RunNextTaskTaskListener(_taskRunner, runTask,
                     runCompletionListener);
 
+            final TaskListener consumerInitFinishedListener = new RunNextTaskTaskListener(_taskRunner, new Task() {
+                @Override
+                public void execute() throws Exception {
+                    getAnalysisListener().rowProcessingBegin(_analysisJob, rowProcessingMetrics);
+                    for (RowProcessingConsumer rowProcessingConsumer : getConsumers()) {
+                        final Collection<ActiveOutputDataStream> activeOutputDataStreams = rowProcessingConsumer.getActiveOutputDataStreams();
+                        for (ActiveOutputDataStream activeOutputDataStream : activeOutputDataStreams) {
```

Comment:
I guess this should be recursive somehow, because right now I think it will then only cover child-streams, but not children-of-children-streams.